src/Native/Runtime/coreclr/poppack.h

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is this file placed to src/Native/Runtime/coreclr but the pshpack4.h to src/Native/Runtime?
It seems that a better place for all of these would be src/Native/inc/unix.